src/transformers/models/jina_bert/modular_jina_bert.py

```diff
@@ -0,0 +1,1169 @@
+import numpy as np
+from torch import nn
+
+
+# TODO this import is not copied to modeling_jina_bert.py
+try:
+    from torch.nn.functional import scaled_dot_product_attention
+except ImportError:
+    scaled_dot_product_attention = None
+
+# TODO check
+# This is used by encode but user may not have it installed
+try:
+    from tqdm.autonotebook import trange
```

Comment:
`tqdm` is used in the [original implementation](https://huggingface.co/jinaai/jina-bert-v2-qk-post-norm/blob/main/modeling_bert.py#L1162) but I think we can just remove it.